src/styles/_catalog.scss

```diff
@@ -1,22 +1,45 @@
+$card-height: 300px;
+$card-width: 200px;
+$gap-size: 48px;
+$catalog-max-width: $card-width * 4 + $gap-size * 3;
+$text-hover-color: #34568b;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  justify-items: center;
+  align-items: center;
```

Comment:
Do you need all this rules?

Reply:
@MaksimusCRS yeah,  ` justify-content: center;` puts my grid in the center of the screan if it wider than the grid width, but not put it on the left side)
and `justify-items: center;` makes the same for each card, if screan includes 3 card only you will see it on the left side but  not on the center)

Reply:
@MaxManis OK, and what about align-content, why do you need it?)

Reply:
@MaksimusCRS yeah, agreed. `align-*` rules doesnt really helpful here because of the same sizes for row and content)